components/script/dom/webglrenderingcontext.rs

```diff
@@ -1281,15 +1290,26 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             return self.webgl_error(InvalidOperation);
         }
 
-        if count <= 0 {
-            return self.webgl_error(InvalidOperation);
+        if count < 0 {
+            return self.webgl_error(InvalidValue);
         }
 
         if offset < 0 {
             return self.webgl_error(InvalidValue);
         }
 
-        if self.current_program.get().is_none() || self.bound_buffer_element_array.get().is_none() {
+        if self.current_program.get().is_none() {
```

Comment:
This is also wrong, from the notes section of https://www.khronos.org/opengles/sdk/docs/man/xhtml/glDrawElements.xml :
> If the current program object, as set by glUseProgram, is invalid, rendering results are undefined. However, no error is generated for this case.

So a quote to the spec, and maybe just return here in the `current_program` or `bound_buffer_element_array` are none? (And probably do the same on overflow?)

Probably worth checking what other implementors or drivers do.